packages/charts/src/charts/line/line-chart.component.tsx

```diff
@@ -121,6 +134,9 @@ export const LineChart: FC<Props> = ({
     >
       <Grid xScale={xScale} yScale={yScale} />
       <Axes xScale={xScale} yScale={yScale} />
```

Comment:
Instead of implementing it in all of charts - with axes - please extend the `Axes` component to support titles.